CorsixTH/Lua/config_finder.lua

```diff
@@ -488,6 +491,26 @@ audio_music = nil -- [[X:\ThemeHospital\Music]]
 --
 soundfont = nil -- [[X:\ThemeHospital\FluidR3.sf3]]
 
+-------------------------------------------------------------------------------
+-- Midi API and Device settings.
+-- By default, CorsixTH uses FluidSynth or build defined MIDI synthesizer.
+-- You can change the API to target other available MIDI backends using these
+-- settings on supported platforms.
+-- Possible values for midi_api are:
+--   <nil>      - Uses SDL_Mixer's MIDI backend, typically FluidSynth
+--   Native     - Use any available platform MIDI API
+--   ALSA       - Use the ALSA MIDI API (Linux only)
+--   JACK       - Use the JACK MIDI API (Unix-like systems with JACK)
```

Comment:
@colinjmatt our Appimage seems to try to latch to JACK or ALSA in some circumstances (and notably throws warnings that have no impact). Any impact here?